cookbooks/continental_compression_with_imposed_faults/continental_compression_with_imposed_faults.prm

```diff
@@ -0,0 +1,446 @@
+    set Variable names = x,y
+    set Function constants = h=400e3, ts1=273, ts2=683, ts3=993, ts4=1573,\
+                             A1=1.e-6, A2=0.25e-6, A3=0.0, A4=0.0,\
+                             k1=2.5, k2=2.5, k3=2.5, k4=2.5, \
+                             qs1=0.06125, qs2=0.04125, qs3=0.03625, qs4=0.03625
+    set Function expression = if( (h-y)<=20.e3, \
+                                  ts1 + (qs1/k1)*(h-y) - (A1*(h-y)*(h-y))/(2.0*k1), \
+                                  if( (h-y)>20.e3 && (h-y)<=40.e3, \
+                                      ts2 + (qs2/k2)*(h-y-20.e3) - (A2*(h-y-20.e3)*(h-y-20.e3))/(2.0*k2), \
+                                      if( (h-y)>40.e3 && (h-y)<=80.e3, \
+                                          ts3 + (qs3/k3)*(h-y-40.e3) - (A3*(h-y-40.e3)*(h-y-40.e3))/(2.0*k3), \
+                                          ts4 + ((h-y-80.e3)/1.e3)*0.3 ) ) );
+  end
+end
+
+# Constant internal heat production values (W/m^3) for background material
+# and compositional fields.
+subsection Heating model
+  set List of model names = compositional heating, adiabatic heating, shear heating
+  subsection Compositional heating
+    set Use compositional field for heat production averaging = 1, 0, 0, 0, 1, 1, 1,1
+    set Compositional heating values = 0., 0., 0., 0., 1.0e-6, 0.25e-6, 0.0, 0.0
+  end
+  subsection Adiabatic heating
+    set Use simplified adiabatic heating = true
+  end
+end
+
+# Material model
+# Rheology: Non-linear viscous flow and Drucker Prager Plasticity
+# Values for most rheological parameters are specified for a background material and
+# each compositional field.  Values for viscous deformation are based on dislocation
+# creep flow-laws, with distinct values for the upper crust (wet quartzite), lower
+# crust (wet anorthite) and mantle (dry olivine).
+subsection Material model
+  set Model name = visco plastic
+
+  set Material averaging = harmonic average only viscosity
+
+  subsection Visco Plastic
+
+    # Reference temperature and viscosity
+    set Reference temperature = 273
+
+    # The minimum strain-rate helps limit large viscosities values that arise
+    # as the strain-rate approaches zero.
+    # The reference strain-rate is used on the first non-linear iteration
+    # of the first time step when the velocity has not been determined yet.
+    set Minimum strain rate = 1.e-20
+    set Reference strain rate = 1.e-16
+
+    # Limit the viscosity with minimum and maximum values
+    set Minimum viscosity = 1e18
+    set Maximum viscosity = 1e26
+
+    # Thermal diffusivity is adjusted to match thermal conductivities
+    # assumed in assigning the initial geotherm
+    set Define thermal conductivities = true
+    set Thermal conductivities        = 2.5
+    set Heat capacities               = 750.
+
+    #The densities below are set to achieve the following densities at the reference temperature (273 K):
+    set Densities                     = background: 3216.374269005848, \
```

Comment:
@PrajaktaPMohite - Please still add in the following documentation line below line 298: `# upper_crust - 2800, lower_crust - 2900, background/mantle_lithosphere/asthenosphere - 3300`.